e2e/geomap-suite/geomap-layer-types.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { e2e } from '@grafana/e2e';
+
+const DASHBOARD_ID = 'P2jR04WVk';
+
+e2e.scenario({
+  describeName: 'Geomap layer types',
+  itName: 'Tests changing the layer type',
+  addScenarioDataSource: false,
+  addScenarioDashBoard: false,
+  skipScenario: false,
+  scenario: () => {
+    e2e.flows.openDashboard({ uid: DASHBOARD_ID, queryParams: { editPanel: 1 } });
+    cy.get('[data-testid="layer-drag-drop-list"]').should('be.visible');
+    e2e.components.PanelEditor.OptionsPane.fieldLabel('Map layers Layer type').should('be.visible');
+    cy.get('[data-testid="layer-drag-drop-list"]').contains('markers');
+    e2e().wait(500);
```

Comment:
are these `wait` calls necessary? There may be something like `await` similar to Jest's `waitFor` function in cypress

Scaled out this would drastically increase our e2e build step time